tests/spec/core/data-tests-spec.js

```diff
@@ -63,5 +68,33 @@ describe("Core — data-tests attribute", () => {
         expect(pathname.endsWith(`test${i}.html`)).toBe(true);
       });
     });
+    it(`adds emojis for manual and https related tests`, () => {
+      
+      let test = doc.querySelector("#metadata-tests ul").querySelectorAll("li")
+
+      // for test 1
+      expect(test[0].children[0].innerText.trim()).toEqual("this-is-secure");   // filename
```

Comment:
might be better here:

```JS
const filename = li.innerText.trim();
expect(filename).toEqual("this-is-secure");
```

Then it's much more clear exactly what you are testing, same applies to the ones below. Basically, if you find that you have to leave a comment, e.g., `// filename`, that's a strong sign you should be using a variable.

Reply:
note also here, the fact that you are calling `.trim()` seems like it might be a bug in the implementation. You shouldn't need to call trim, as the implementation should have trimmed it.